src/bundle/Resources/public/scss/_filters.scss

```diff
@@ -78,7 +78,6 @@
         padding: calculateRem(8px) calculateRem(48px) calculateRem(8px)  calculateRem(8px);
         display: inline-block;
         position: relative;
-        height: calculateRem(38px);
```

Comment:
maybe just `min-height` instead?

Reply:
Do we need `min-height`? You would like to have `min-height` because it will be then more error prone?

Reply:
It was just a question. Maybe you wanted the element to be at least 38px tall, am I right?